library/ssl_tls.c

```diff
@@ -5784,10 +5784,10 @@ const char *mbedtls_ssl_get_version( const mbedtls_ssl_context *ssl )
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
 const char* mbedtls_ssl_get_key_exchange_name( const mbedtls_ssl_context* ssl ) {
```

Comment:
This function is called from `ssl_client2` _after_ the handshake, in which case it will return `NULL` now. 

As this is for debugging only, I would suggest to remove it altogether for now.